lib/fix_html.js

```diff
@@ -56,17 +58,60 @@ function fixReferences ($, fname, sources, files, page) {
       anchor = m[2]
     }
 
-    // Get the target source file it points to (eg, `docs/usage.md`).
-    var target = getTarget(origUrl, base)
-
-    // Ensure that it's available.
-    if (!sources[target]) {
-      throw new Error(`${base}: Unknown reference '${origUrl}'`)
+    // if absolute markdown file, make link relative to MD file
+    if (page.absolute) {
```

Comment:
Currently this isn't tested. If you are still interested, do you think you could add a test? I rebased your branch and updated the tests to pass, but this brings down the coverage too much.